en/Releases/enebular/2.4.1.md

```diff
@@ -0,0 +1,23 @@
+---
+lastUpdated: 2018-08-03
+---
+
+# 2.4.1 (August 3rd, 2018)
+
+## New
+
+None
+
+## Fixed
+
+- Fixed a bug that no e-mail was sent even when creating an account from new registration.
```

Comment:
Fixed a bug that failed to send users an email when creating a new account.

Reply:
Thank you so much